test/bin/scenario.sh

```diff
@@ -196,6 +203,9 @@ prepare_kickstart() {
               -e "s|REPLACE_REDHAT_AUTHORIZED_KEYS|${REDHAT_AUTHORIZED_KEYS}|g" \
               -e "s|REPLACE_PUBLIC_IP|${PUBLIC_IP}|g" \
               -e "s|REPLACE_FIPS_COMMAND|${fips_command}|g" \
+              -e "s|REPLACE_ENABLE_MIRROR|${ENABLE_MIRROR}|g" \
+              -e "s|REPLACE_HOSTNAME|${hostname}|g" \
```

Comment:
Can we call it `REPLACE_MIRROR_HOSTNAME`?

Reply:
Done